### PR TITLE
SISRP-16892 - Fluid - create css class uc-status-message to style the…

### PR DESCRIPTION
**Pull request by marcpechaitis**
… status message displayed on the Fluid pages

https://jira.berkeley.edu/browse/SISRP-16892

### CODE REVIEW

**Comment by marcpechaitis** [comment]
@christianvuerings I'm fixing build errors and will resubmit

**Comment by marcpechaitis** [comment]
@christianvuerings please review

File: src/css/cs/items/_kitchensink.scss [hunk 1]
```diff
@@ -376,6 +376,19 @@
...
   }
 
+  // Style the text that appears in the HTMLArea on UC_STATUS_MSG_SBF
+  .uc-status-message {
+    border: 0;
+    box-shadow: none;
+    font-size: 16px;
+    font-weight: bold;
+    margin: 0;
+    outline: none;
```
Comment (christianvuerings) [inline src/css/cs/items/_kitchensink.scss:386]: @marcpechaitis not a blocker but would suggest seeing whether we can disabe this to make it even more accessible (this allows people to see where the focus is)

Comment (marcpechaitis) [inline src/css/cs/items/_kitchensink.scss:386]: @christianvuerings I'm not sure what you're referring to "this"?  The whole class?

"this allows people to see where the focus is" - do you mean the blue highlight around the textarea?  That is disabled by `outline: none`

Comment (marcpechaitis) [inline src/css/cs/items/_kitchensink.scss:386]: the page behaves the same to the non-screen-reading user as before...at least in my experience

Comment (christianvuerings) [inline src/css/cs/items/_kitchensink.scss:386]: @marcpechaitis indeed, it's about the `outline: none` - http://www.outlinenone.com/ http://stackoverflow.com/a/19455695/117193

Comment (marcpechaitis) [inline src/css/cs/items/_kitchensink.scss:386]: OK I understand reasons why `outline: none` is a bad idea...I am going to remove it

**Pull request merged**